avro/src/test/scala/test/AvroRandomGeneratorSpec.scala

```diff
@@ -0,0 +1,58 @@
+package com.memsql.spark.examples.avro
+
+import org.scalatest._
+import org.apache.avro.Schema
+import org.apache.avro.generic.GenericData
+
+class AvroRandomGeneratorSpec extends FlatSpec {
+  "AvroRandomGenerator" should "create Avro objects with random values" in {
+    val avroJsonSchemaStringTest = s"""
```

Comment:
I'm trying to avoid complex types in general in this diff. The reason Record is included is because that is the top level object.